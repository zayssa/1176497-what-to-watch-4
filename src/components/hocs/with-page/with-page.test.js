import React from "react";
import renderer from "react-test-renderer";
import withPage from "./with-page.jsx";
import MockComponent from "../../../mocks/mock-component.jsx";

const MockComponentWrapped = withPage(MockComponent);

describe(`WithActiveItem snapshot`, () => {
  it(`should render WithActiveItem component`, () => {
    const tree = renderer
      .create(
          <MockComponentWrapped />
      ).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
