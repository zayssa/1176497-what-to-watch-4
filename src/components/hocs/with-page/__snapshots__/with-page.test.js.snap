// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`WithActiveItem snapshot should render WithActiveItem component 1`] = `<div />`;
